public/index.html

```diff
@@ -0,0 +1,84 @@
+<!DOCTYPE html>
+<html>
+  <head>
+      <meta charset="utf-8">
```

Comment:
Some of your HTML is working on 4-space indentation, and that should be 2-space.